bears/python/PythonPackageInitBear.py

```diff
@@ -0,0 +1,41 @@
+import os
+import re
+
+from coalib.results.Result import Result
+from coalib.bears.GlobalBear import GlobalBear
+
+
+class PythonPackageInitBear(GlobalBear):
+    """
+    Looks for missing __init__.py files in directories containing python files.
+    """
+
+    def run(self,
+            ignore_dirs: list = None,
+            create_empty_init_file: bool = True):
+        """
+        :param root_directory:
+            The root directory of the project whose tree has to be checked
+        :ignore_dirs:
+            List of directories to be ignored
+        """
+        for files in sorted(self.file_dict):
+            self.debug('FILE: ', files)
+        for file_path in self.file_dict:
+            dir_path = os.path.split(file_path)[0]
+            ignore = False
+            for ignore_dir in ignore_dirs if ignore_dirs else []:
+                if dir_path.endswith(ignore_dir):
+                    ignore = True
+                    break
+            if ignore:
+                continue
+            if not "__init__.py" in os.listdir(dir_path):
```

Comment:
Also Sils requests you check this from the file_dict instead of `os.listdir()`
>IMO it should check if the init is in the file dict, not on the FS, so we behave cleanly